transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -170,6 +170,12 @@ protected void wakeup(boolean inEventLoop) {
         }
     }
 
+    @Override
```

Comment:
Add a bit of javadoc about  the meaning of the returned boolean? (especially as we no longer delegate a call to the queue which could have effectively provided that information via its javadoc). Ditto the other `removeTask` methods.

Reply:
@davidmoten sure..

Reply:
ah I meant what does true mean and what does false mean. Does true mean it was found on the queue and removed? Does false mean it is either not supported or not found on the queue?